common/testParameters.js

```diff
@@ -22,6 +22,19 @@ export default _.filter((t) => t, [{
 }, {
   line: getLineNumber(),
 
+  previousConformedValue: '$111,222',
+  previousPlaceholder: '$___,___',
+  rawValue: '$111,22',
+  mask: ['$', /\d/, /\d/, ',', /\d/, /\d/, /\d/],
+  currentCaretPosition: 5,
+
+  conformedValue: '$11,122',
+  adjustedCaretPosition: 5,
+
```

Comment:
I've added a failing test that passes with `core-v4.0.0`.  Basically if you have 3 of the same chars in a row, then your "track right char" method fails.